<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExportDialog</class>
 <widget class="QWidget" name="FileExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Structures</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="file_format_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="filename_lbl">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filename_le"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="format_lbl">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="format_combo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="export_btn">
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
